build/media-suite_compile.sh

```diff
@@ -1825,13 +1825,21 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
     fi
 
     if ! mpv_disabled vulkan; then
+        # Get the known_good json and get the commit version of the Vulkan-Headers.
+        # It is assumed that the version number has quotes around them (sed)
+        [[ "$jq" != y ]] && do_pacman_install jq
+        _Vulkan_Version=$(
+            ${curl_opts[@]} https://raw.githubusercontent.com/KhronosGroup/Vulkan-ValidationLayers/master/scripts/known_good.json |
+            jq  '.repos[1].commit' |
```

Comment:
You can use `-r` to output raw strings.

Reply:
Oh.

Reply:
I wasn't sure what it meant by raw instead of json

Reply:
`jq --raw-output '.repos[] | select(.name == "Vulkan-Headers").commit'` should supposedly work, not apparently not in Windows.